Translations/UltraPermissions_Dutch.lang

```diff
@@ -1,172 +1,172 @@
-#GUI_PERMEDITOR_AVAILABILITY_ONLYTHISSERVER: Only available on **this** Server
-#GUI_PERMEDITOR_AVAILABILITY_SPECIFICSERVER: Available on **%1** Server
-#GUI_PERMEDITOR_ADD_LEFTACTION: **Left Click** to add Permission
-#GUI_PERMEDITOR_ADD_RIGHTACTION: **Right Click** to add **Bungee Permission**
-#GUI_PERMEDITOR_ADD_TYPEIN: Type in a permission
-#GUI_PERMEDITOR_EDIT_TITLE: Edit %1
-#GUI_PERMEDITOR_EDIT_ACTION: Click to edit **%1**
-#GUI_PERMVIEW_EDITPERM_TITLE: Edit Permission
-#GUI_PERMVIEW_EDITPERM_ACTION: Click to edit Permission
-#GUI_PERMVIEW_EDITPERM_TYPEIN: Type in the new permission
-#GUI_PERMVIEW_COPY_ACTION: Click to select a **target**
-#GUI_PERMVIEW_NOMULTINAMEEDIT: you cannot rename multiple permissions at once
-#GUI_STORAGESETTINGS_TITLE: Storage Settings
-#GUI_STORAGESETTINGS_TYPEINWORLD: Type in a world name
-#GUI_STORAGESETTINGS_NOWORLD_ACTION: Click to **restrict** this group to a specific World
-#GUI_STORAGESETTINGS_WORLD_ACTION: Click to allow this group in all Worlds
-#GUI_STORAGESETTINGS_NOSERVER_ACTION: Click to **restrict** this group to this Server
-#GUI_STORAGESETTINGS_SERVER_ACTION: Click to allow this group on all Servers
-#GUI_STORAGESETTINGS_RESTRICTED_TO: Restricted to **%1**
-#GUI_PASTESELECTION_PASTE_ACTION: Click to **paste** %1
-GUI_PERMEDITOR_MANUAL_TITLE: Typ in Chat
-GUI_PERMEDITOR_MANUAL_ACTION: **Klik** om de machtiging te typen
-GUI_PERMEDITOR_MANUAL_DIALOG_SUBTITLE: Typ een machtiging
-GUI_INHERITVIEW_ADD_ACTION: Klik om **toe te voegen**
-GUI_INHERITVIEW_ADD_EXPLANATION: Hiermee worden alle machtigingen toegevoegd van **%1** tot **%2**
-GUI_INHERITVIEW_REMOVE_ACTION: Klik om **te verwijderen**
-GUI_INHERITVIEW_REMOVE_EXPLANATION: Alle machtigingen van **%1** worden toegevoegd aan **%2**
-GUI_CHANGEORDER_TITLE: Volgorde bewerken
-GUI_CHANGEORDER_INCREASE: **Klik met de rechtermuisknop** om te vergroten
-GUI_CHANGEORDER_DECREASE: **Klik met de linkermuisknop** om te verlagen
-GUI_PERMISSIONLOG_ACTION: Klik om deze toestemming toe te voegen aan een **groep** of **gebruiker**
-GUI_PERMISSIONLOG_RESULT: Resultaat: %1
-GUI_SETTINGS_FORMAT_NAME: Chat- en tablijstindeling
-GUI_SETTINGS_MISC_NAME: Diversen
-EDITING_PREFIX_EDIT: **Voorvoegsel bewerken
-EDITING_PREFIX_TYPEIN: Typ een nieuw voorvoegsel
-EDITING_PREFIX_TYPEINNONE: Typ **geen** om geen voorvoegsel te hebben
-EDITING_SUFFIX_EDIT: **Achtervoegsel bewerken
-EDITING_SUFFIX_TYPEIN: Typ een nieuw achtervoegsel
-EDITING_SUFFIX_TYPEINNONE: Typ **geen** om geen achtervoegsel te hebben
-CREATE_GROUP_TITLE: Groep maken
-CREATE_GROUP_SUBTITLE: Voer een groepsnaam in
-CREATE_GROUP_NOSPACES: De groepsnaam mag geen spaties bevatten
-CREATE_GROUP_NOCOLORS: Gebruik geen chatkleuren in de groepsnaam
-CREATE_GROUP_NAMETAKEN: Deze groepsnaam is al in gebruik
-RENAME_GROUP_TITLE: Naam van groep wijzigen
-RENAME_GROUP_SUBTITLE: Voer een nieuwe naam in
-CONFIRM_DELETION_OF: Bevestig verwijdering van %1
+REPORT: Aangeven		PERMISSION: Machtiging
```

Comment:
This line till line 58 has multiple phrases in one line. Also there are several phrases that don't belong to UltraPermissions.

Reply:
not sure hop that happened